src/app/routes/utils/regex/utils/index.js

```diff
@@ -6,7 +6,7 @@ const legacyAssetUriRegex =
 const variantRegex = '/simp|/trad|/cyr|/lat';
 const articleLocalRegex = 'articles|erthyglau|sgeulachdan';
 const mediaIdRegex = '[a-z0-9]+';
-const mediaServiceIdRegex = 'bbc_[a-z]+_radio|bbc_[a-z]+_tv';
+const radioServiceIdRegex = 'bbc_[a-z]+_radio';
```

Comment:
Not sure if this is the right place, but I'd be keen to clean up this misuse of the `serviceId` name where possible

`bbc_korean_id` isn't a service id - it is a master brand

Outside the radio code, service id is usually understood to be the world service name, like "korean" and "pidgin" - master brand is much less ambiguous and aligns with product / editorial naming

Reply:
I'll update!

Reply:
Done